warehouse/forklift/legacy.py

```diff
@@ -948,8 +948,8 @@ def file_upload(request):
         if request.user:
             msg = (
                 (
-                    "The user '{0}' isn't allowed to upload to project '{1}'. "
-                    "See {2} for more information."
+                    "The user '{}' isn't allowed to upload to project '{}'. "
+                    "See {} for more information."
```

Comment:
Should this become an f-string? I'm surprised it didn't catch this use of `.format`.

Reply:
I suspect it has something to do with the complex usage of `request.help_url(_anchor="project-name"),` and quotations - there's a test that shows this should be a noop: 
https://github.com/asottile/pyupgrade/blob/ff43f9a6ab4c37d0a89a4bb5ae338c2d0233b5eb/tests/features/fstrings_test.py#L34-L37